### PR TITLE
Support Active Support 5.2

### PR DESCRIPTION
**Pull request by sgrif**
cc @Shopify/rails

### CODE REVIEW

File: active_shipping.gemspec [hunk 1]
```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
 
 
   s.add_dependency("measured", ">= 2.0")
-  s.add_dependency("activesupport", ">= 4.2", "< 5.2.0")
+  s.add_dependency("activesupport", ">= 4.2", "<= 5.2.0.beta2")
```
Comment (rafaelfranca) [inline active_shipping.gemspec]: Maybe it is better to remove the upper restriction like we are doing in the projects in the rails organization.

Comment (rafaelfranca) [reply active_shipping.gemspec]: BTW, beta2 is already allowed by `< 5.2.0` since for rubygems any pre release is less than the final release.

**Comment by Edouard-chin** [comment]
Last previous broken build had 4 failures on 5.0, there is 5 failures on 5.1 and 5.2, the  extra one is `test_domestic_combined_response` and doesn't seems to be related so LGTM.

**Review by Edouard-chin — APPROVED** [review]

File: active_shipping.gemspec (hunk 1 above)
Comment (sgrif) [reply active_shipping.gemspec]: I'm not fond of that policy personally, and yeah I realized this is mostly pointless until 5.2 final is out

**Comment by lbrito1** [comment]
Hi,
Even though this moves forward, won't we need to update [active_utils](https://github.com/shopify/active_utils) as well? Currently it depends on activesupport < 5.2.0.

**Comment by Edouard-chin** [comment]
#556

**Pull request closed without merging**